addons/medical/XEH_postInit.sqf

```diff
@@ -291,3 +291,9 @@ if (hasInterface) then {
         [player] call FUNC(init);
     }] call CBA_fnc_addEventHandler;
 };
+
+if (["ACE_Arsenal"] call EFUNC(common,isModLoaded)) then {
```

Comment:
There is no need for this, the event simply won't be ran if ACE Arsenal is not loaded.

Reply:
Are you sure I should remove it? everywhere else the check is there, it's also a good idea to not add Ehs that won't run anyway.

Reply:
Hmh, it's not used anywhere where adding EHs. I don't know about the performance impact, I am assuming it's basically negligible. @commy2 ?